test_site/sass/blog.scss

```diff
@@ -1,5 +1,21 @@
+@use 'zola';
+@use 'sass:map';
+
 body {
-  background: red;
+  background: 
+    // once we are using a more contemporary version of Dart/Sass we
+    // can use notation like:
+    //   map.get(zola.$config, "extra", "sass", "background_color")
```

Comment:
Are we sure it's not supported yet? This seems..... very bad to use.

Reply:
fixed in grass in the following PR

https://github.com/connorskees/grass/pull/83

Reply:
i could patch in the latest grass release (1.13.0) in this PR, or i could do it as a follow up.  preference?